README.md

```diff
@@ -1,51 +1,58 @@
+
+Once the plugin is installed and enabled, you will see a new **Site Search** entry in the admin menu. Go to this entry to configure the Site Search plugins :
+
+  1. Enter your Swiftype API key on the first screen.
+  3. Name your search engine and optionnaly choose a language.
+  3. Build your search index by clicking the "Synchronize" button.
+
+**Note :**
+  * Using this plugin assumes that you have already created a Site Search account on https://swiftype.com/.
+  * If you already have an account, you will need your API Key in order to configure the plugin. You can find it in on top of the Site Search [Account Settings](https://app.swiftype.com/settings/account) screen on the Site Search site.
 
 ## Usage
 
 For additional information on how to use and extend the plugin, please
 visit the [plugin notes
 page](https://wordpress.org/plugins/swiftype-search/other_notes/).
 
-## Development
+## FAQ 🔮
 
-### Running unit tests
+### Where do I report issues with the plugin?
 
-To run the unit tests locally, first install the WordPress unit testing framework and a local copy of WordPress by running `scripts/install-wp-tests.sh <db name> <db user name> <db password>`.
+If something is not working as expected, please open an [issue](https://github.com/swiftype/swiftype-wordpress/issues/new).
 
-Next, run the tests with `scripts/run_tests.sh`. This runs the non-Multisite tests against the version of WordPress you installed in the previous commands.
+### Where else can I go to get help?
 
-When new commits are pushed, the tests will be run automatically on Swiftype's CI server using several different versions of PHP and WordPress (see `scripts/ci_build.sh`). We are working on making these test runs public.
+You can checkout the [Site Search Plugin discuss forum](https://wordpress.org/support/plugin/swiftype-search/).
 
+You can contact our support by sending an email to support@swiftype.com.
 
-### Version Tagging
+## Contribute 🚀
 
-1. Find next version using `git tag --list`
-2. Change version number in `README.txt` (1 occurrence) and `swiftype.php` (2 occurrences)
-3. Update changelog in `README.txt`
-4. Commit updates `git commit -am "bump version"`
-5. `git push`
-4. Tag version `git tag v1.x.yz`
-5. `git push --tags`
-6. Publish to WP `./scripts/publish.sh 1.x.yz`
+We welcome contributors to the project. Before you begin, a couple notes...
 
++ Before opening a pull request, please create an issue to [discuss the scope of your proposal](https://github.com/swiftype/swiftype-wordpress/issues).
++ Please write simple code and concise documentation, when appropriate.
 
-### Pre-commit hook
+## License 📗
 
-There is a pre-commit hook to automatically concatenate and minify JavaScript every time you commit. This is done with [Grunt](http://gruntjs.com/).
+[Apache 2.0](https://github.com/swiftype/swiftype-wordpress/blob/master/LICENSE) © [Elastic](https://github.com/elastic)
```

Comment:
The license file above in MIT.  I think A2 is correct.

Reply:
Will change the licence file to an Apache 2.0 licence which is the default at Elastic.

Reply:
The License file is now A2 (ce3e612).

Warning: the link into the README point to the master branch where the file is still the old one.
It will be correct when we will have merged this branch into master.

Reply:
Since you're updating license related stuff, you should add in the NOTICE file from this PR: https://github.com/swiftype/swiftype-wordpress/pull/41/files#diff-d8b7f7a4958ed5af485670e07cf80cac, and then close that PR as it will be obsolete.

Reply:
The NOTICE file is already in the PR (actually I have merged the PR #41 in mine and change few minor things after).

I have close the PR #41 for being obsolete.